tcms/locale/en/LC_MESSAGES/django.po

```diff
@@ -571,18 +572,18 @@ msgstr ""
 msgid "New Test Plan"
 msgstr ""
 
-#: tcms/settings/common.py:297 tcms/testplans/templates/testplans/get.html:130
+#: tcms/settings/common.py:297
+#: tcms/testcases/templates/testcases/mutable.html:12
+#: tcms/testplans/templates/testplans/get.html:130
 msgid "New Test Case"
 msgstr ""
 
-#: tcms/settings/common.py:299
+#: tcms/settings/common.py:299 tcms/testplans/templates/testplans/get.html:120
+#: tcms/testruns/templates/testruns/get.html:211
+#: tcms/testruns/templates/testruns/mutable.html:11
```

Comment:
> __[Secret Scan]__ It may be dangerous to commit the AWS secret access key. ([view](https://sider.review/gh/repos/92381207/pulls/2528#issue-6063097))
>
> <details>
>
> AWS secret access key is an important credential that allows you to use AWS programmatically.
> Once this leaks, an attacker can access and operate your AWS resources.
>
> See also:
> * https://docs.aws.amazon.com/general/latest/gr/aws-access-keys-best-practices.html
>
> | Rule | Severity |
> | ---- | -------- |
> | `sider.secrets.aws.secret_access_key` | `warning` |
>
> You can [close this issue](https://sider.review/gh/repos/92381207/pulls/2528#issue-6063097) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

I closed this issue. Reason:
- When this is not an AWS secret access key.